icaruscode/Decode/DecoderTools/TriggerDecoderV2_tool.cc

```diff
@@ -685,7 +683,7 @@ namespace daq
     // absolute time trigger (raw::ExternalTrigger)
     //
     fTrigger->emplace_back
-      (fTriggerExtra->triggerID, fTriggerExtra->triggerTimestamp);
+      (fTriggerExtra->triggerID, artdaq_ts);//fTriggerExtra->triggerTimestamp);
```

Comment:
What is the intention in this change?

Reply:
I was having problems with this variable fTriggerExtra->triggerTimestamp producing an empty value here. Can revert this if I have to